src/main/scala/higherkindness/rules_scala/workers/zinc/compile/AnalysisStore.scala

```diff
@@ -107,14 +122,16 @@ class AnxAnalysisStore(files: AnalysisFiles, analyses: AnxAnalyses) extends Anal
 
 class AnxAnalyses(format: AnxAnalysisStore.Format) {
   private[this] val mappers = AnxMapper.mappers(Paths.get(""))
-  private[this] val reader = new ProtobufReaders(mappers.getReadMapper, schema.Version.V1)
+  private[this] val reader = new ProtobufReaders(mappers.getReadMapper, Schema.Version.V1_1)
   private[this] val writer = new ProtobufWriters(mappers.getWriteMapper)
 
   def sortProjection(value: Projection): Projection = {
-    value.copy(
-      id = value.id,
-      prefix = value.prefix,
-    )
+    // I copied this over from the old code, but I'm pretty sure this isn't doing anything useful?
```

Comment:
Can't use copy because it is a Java object rather than a Scala object. I don't think it's doing anything useful, but I want to remove it in another, smaller change, so if it breaks something it is easier to figure out that removing this broke it.